charms/argo-controller/src/components/pebble_component.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+import logging
+
+from charmed_kubeflow_chisme.components.pebble_component import PebbleServiceComponent
+from ops.pebble import Layer
+
+logger = logging.getLogger(__name__)
+
+ARGO_CONTROLLER_CONFIGMAP = "argo-workflow-controller-configmap"
+EXECUTOR_IMAGE_CONFIG_NAME = "executor-image"
+LIVENESS_PROBE_PORT = "6060"
+METRICS_PORT = "9090"
+LIVENESS_PROBE_PATH = "/healthz"
+LIVENESS_PROBE_NAME = "argo-controller-up"
+
+
+class ArgoControllerPebbleService(PebbleServiceComponent):
+    def __init__(
+        self,
+        *args,
+        **kwargs,
+    ):
+        """Pebble service container component in order to configure Pebble layer"""
```

Comment:
May be this doc string should be on line 19?

Reply:
Updated